scripts/ci/automation_full_test.py

```diff
@@ -441,13 +446,13 @@ def get_pipeline_result(test_result_fp, pipeline_result):
                 for failure in failures:
                     message = failure.attrib['message'].replace('\n', '<br>')
                     break
-                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details']:
+                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details'][0]['Details']:
                     if i['Module'] == module:
                         i['Status'] = 'Failed'
                         i['Content'] = build_markdown_content(state, test_case, message, line, i['Content'])
                         break
             else:
-                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details']:
+                for i in pipeline_result[unique_job_name]['Details'][0]['Details'][0]['Details'][0]['Details']:
```

Comment:
What's the purpose to add `[0]['Details']`?

Reply:
![image](https://user-images.githubusercontent.com/18628534/236384758-3c601e6d-9c3a-46f4-bd60-065e58510448.png)
This is due to the data structure we worked out with the azure-powershell team.

Reply:
So CLI only has one element for each `Details` property, while there will be more than one element for `Details` property in azure PS?

Reply:
Yes, there are multiple elements for Details property in azure PS.
Please note that we also have multiple elements in the innermost layer. 
The reason why the other layers only have a single element is that we use multiple instance tests. 
If these tests are run on the same machine, then our details will also have multiple elements.